public/index.html

```diff
@@ -34,19 +34,19 @@
     -->
     <title>%REACT_APP_TITLE%</title>
 
-    <script type="text/javascript" src="%PUBLIC_URL%/vendor/jquery.min.js"></script>
     <script type="text/javascript" src="%PUBLIC_URL%/arethusa/arethusa.packages.min.js"></script>
     <script type="text/javascript" src="%PUBLIC_URL%/arethusa/arethusa.min.js"></script>
     <script type="text/javascript" src="%PUBLIC_URL%/arethusa/arethusa.widget.loader.js"></script>
     <script type="text/javascript">
       window.i18npath = "%PUBLIC_URL%/arethusa/i18n/";
-      window.dagred3path = "vendor/dagre-d3.min.js";
+      window.dagred3path = "vendor/dagre-d3/dagre-d3.min.js";
     </script>
 
-    <link rel="stylesheet" type="text/css" href="%PUBLIC_URL%/arethusa/css/arethusa.min.css">
+    <link rel="stylesheet" type="text/css" href="%PUBLIC_URL%/arethusa/dist/arethusa.min.css">
```

Comment:
That's a good question. This is the only CSS file in `dist/` and the other ones are in the `css` directory.

It is weird that they are in different directories. Do you think it's worth it to a line in `deploy_widget.sh` that moves it to `css/`?

Reply:
hmm. I guess that might be because the ones in dist are built by the arethusa build process and the others are not. It's probably cleaner for them all to be in one directory but I don't feel super strongly about it.

Reply:
I think that makes sense. I'll merge this now but have a PR for Arethusa tomorrow that updates the deploy script.